src/editor/plugins.js

```diff
@@ -104,6 +107,23 @@ export function getPlaceholderPlugin({ text }) {
     }
   }
 
+  function createPlaceholderNode(text) {
+    let span = document.createElement('span');
+    span.innerHTML = `
+<style>
```

Comment:
can't have `<style>` in `<body>` - shadow roots and templates are excused.

Why does it need to be absolutely positioned at all? Can't you just inject the content? That's why there's nothing to focus on in inline elements (I assume), because abspos gives width 0. Block elements are full width, so doesn't matter.